source/Module/Common/Convert/Data.ps1

```diff
@@ -33,5 +33,8 @@ data exclusionRuleList
         V-42567 = 'McAfee:with the assistance of the System Administrator, review each GUID key's szTaskName'
         V-42572 = 'McAfee:If the ExcludedURLs REG_MULTI_SZ has any entries, and the excluded URLs have not been documented with, and approved by, the ISSO/ISSM/DAA'
         V-14654 = 'McAfee:The GUID of the weekly on-demand client scan task varies by system'
+        V-94509 = 'Vsphere: To be added in next release'
```

Comment:
"Next release" is vague and non-vague at the same time.  Can we use 'To Be added in a future release' or "To be added in version 4.5.0' or whatever the next version is?

Reply:
I like future release vs. specific version.